samples/snippets/README.rst

```diff
@@ -5,7 +5,7 @@ Google Cloud Pub/Sub Python Samples
 ===============================================================================
 
 .. image:: https://gstatic.com/cloudssh/images/open-btn.png
-   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/python-pubsub&page=editor&open_in_editor=samples/snippets/README.rst
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=samples/snippets/README.rst
```

Comment:
Should these be pointing to python-doc-samples? Do we need to change a synthtool config somewhere?